ubuntu_24.04-x86_64-dpdk_22.11/Dockerfile

```diff
@@ -0,0 +1,60 @@
+FROM ubuntu:24.04
+
+ENV DPDK_VERSION=v22.11 \
+  IPSEC_MB_VERSION=v1.1
+
+RUN apt-get update
+
+RUN apt-get install -yy --no-install-recommends \
+        software-properties-common \
+	dirmngr \
+	gnupg-agent
+
+RUN apt-get update --fix-missing
+
+RUN apt-get install -yy \
+  autoconf \
+  automake \
+  ccache \
+  clang \
+  gcc \
+  gcc-10 \
```

Comment:
Required? Ubuntu 24.04 default GCC version is already 13. Same for `libstdc++-10-dev` below.